ProjectStructure.md

```diff
@@ -123,9 +172,98 @@ end
 
 - **TARGET MANAGEMENT**
 
-  This section has not been written yet.
+  Generally we have 3 targets which are *Production, Pre-Production, Staging*. According backend, we should create targets for each environment.  
+
+  For each target we have build flag like `-APPSTORE`, `-PREPROD`. If project has satellite apps, they should have custom flag.
+
+  To add a build flag, developer can add any build flag with opening a specific target's build settings and search "Preprocessor Macros" add build flag on necessary build configurations. 
```

Comment:
It will work what it is expected, but for swift projects, flags (-DAPPSTORE, -DPREPROD) should be under Target->Build Setttings->Custom Flags->Other Swift Flags